x-pack/plugins/case/common/constants.ts

```diff
@@ -14,6 +14,8 @@ export const CASES_URL = '/api/cases';
 export const CASE_DETAILS_URL = `${CASES_URL}/{case_id}`;
 export const CASE_CONFIGURE_URL = `${CASES_URL}/configure`;
 export const CASE_CONFIGURE_CONNECTORS_URL = `${CASE_CONFIGURE_URL}/connectors`;
+export const CASE_CONFIGURE_CONNECTOR_DETAILS_URL = `${CASE_CONFIGURE_CONNECTORS_URL}/{connector_id}`;
+export const CASE_CONFIGURE_PUSH_URL = `${CASE_CONFIGURE_CONNECTOR_DETAILS_URL}/push`;
```

Comment:
I think it will be better to not have a separate route and move the logic of this route insise `api/cases/{id}/_push` that already exists. The advantage of this would be that if the requests extranal service fail the whole request will fail and we will not update the push parameters to the case. At the moment we do two API calls one to the actions API and one to cases. Its a good opportunity to merge them in one. Also, `push` under `configure` does not make much sense imo.